cabal-testsuite/README.md

```diff
@@ -182,24 +188,15 @@ string output test, if that is how your test is "failing."
 Hermetic tests
 --------------
 
-By default, we run tests directly on the source code that is checked into the
-source code repository.  However, some tests require programmatically
-modifying source files, or interact with Cabal commands which are
-not hermetic (e.g., `cabal freeze`).  In this case, cabal-testsuite
-supports opting into a hermetic test, where we first make copy of all
-the relevant source code before starting the test.  You can opt into
-this mode using the `withSourceCopy` combinator (search for examples!)
-This mode is subject to the following limitations:
+Tests are run in a hermetic environment, each test is run in a fresh temporary
```

Comment:
It's not exactly hermetic without a chroot: it can access absolute paths.

Reply:
Very true.